src/rgw/rgw_sal.cc

```diff
@@ -118,7 +118,7 @@ rgw::sal::Driver* DriverManager::init_storage_provider(const DoutPrefixProvider*
   rgw::sal::Driver* driver{nullptr};
 
   if (cfg.store_name.compare("rados") == 0) {
-    driver = newRadosStore(&io_context);
+    driver = newRadosStore(io_context);
```

Comment:
You will have to make the same change in rgw_sal_rados.cc

Reply:
Same question as in your other comment